source/extensions/wasm_runtime/null/BUILD

```diff
@@ -11,6 +11,7 @@ envoy_extension_package()
 envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
+    category = "SOMECAT",
```

Comment:
not seeing any wasm cats

Reply:
@lizan

Reply:
i see this in `extensions_build_config.bzl` and im assuming the cat should be something like `envoy.wasm.runtime` 

im wondering if its ok to use this without it having a factory in source code

Reply:
There is a factory in source/extensions/common/wasm/wasm_runtime_factory.h, I think what you say is right.

Reply:
ah, perhaps my grep command to get the cats wasnt perfect, updating...